roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -201,7 +213,11 @@ class Namespace(object):
 
 
 class ResetNeutron(object):
-    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-proxy', 'neutron-linuxbridge-agent']
+    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-agent', 'neutron-linuxbridge-agent']
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-d', '--device', help='devices to be reset. eg. vxlan or bridge')
```

Comment:
This help message could be improved -- it's currently ambigious if this means a specific device name e.g. brq-00000000-00 or device type e.g. bridge.